src/Bonsai.Lsl/Bonsai.Lsl.csproj

```diff
@@ -38,4 +38,10 @@
     <Content Include="..\liblsl\win-x64\bin\*.dll" PackagePath="runtimes\win-x64\native" CopyToOutputDirectory="PreserveNewest" />
   </ItemGroup>
 
+  <ItemGroup>
+    <Compile Update="Native\LSL.cs">
+      <CopyToOutputDirectory>Always</CopyToOutputDirectory>
+    </Compile>
+  </ItemGroup>
+
```

Comment:
Is this needed? It seems to copy the .cs source file itself to the output directory.